src/roman-numerals/index.js

```diff
@@ -4,6 +4,34 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {let  mainStr = roman.toUpperCase()
+   
+let map = {
+    "I": "1",
+    "V": "5",
+    "X":"10",
+    "L":"50",
+    "C":"100",
+    "D":"500",
+    "M" : "1000"
+}
+
+let sum = 0
+
+for(let j =0; j<mainStr.length; j++){
+    
+    sum += Number(map[mainStr[j]])
+}
+
+        if(mainStr.includes("IV") || mainStr.includes("IX")){
+            sum = sum-2}
+        if(mainStr.includes("XL") || mainStr.includes("XC")){
+            sum = sum-20
+        }
+        if (mainStr.includes("CD") || mainStr.includes("CM")){
+            sum = sum-200
+        }
```

Comment:
These are what's called magic numbers. Why are you subtrating these numbers specifically?

Reply:
The way the code is written is such that all the numbers corresponding to the roman numerals are added. But for the exceptions where if the number comes before it is one less, a subtraction is required for example;
IV  should be equal to 4
but my code will initially run I + V = 6
The if statements run through the code to check if there are any such occurrences and if so a subtraction of 2 is executed 
so that the value is 4.
the same goes for all the other exceptions.